dtoolkit/accessor/dataframe/set_unique_index.py

```diff
@@ -6,6 +6,6 @@
 
 
 @register_dataframe_method
-@doc(klass="DataFrame", alias="df")
+@doc(s_set_unique_index, klass="DataFrame", alias="df")
```

Comment:
to fix
![image](https://user-images.githubusercontent.com/25895405/188303183-0af0a4b1-116a-4bbc-bbd4-0a6ed7fb7a6a.png)